api/swagger/swagger-dev.json

```diff
@@ -24479,6 +24479,376 @@
+              "type": "object",
+              "properties": {
+                "scheduled_backups": {
+                  "type": "array",
+                  "items": {
+                    "description": "ScheduledBackup represents scheduled task for backup.",
+                    "type": "object",
+                    "properties": {
+                      "scheduled_backup_id": {
+                        "description": "Machine-readable ID.",
+                        "type": "string",
+                        "x-order": 0
+                      },
+                      "service_id": {
+                        "description": "Machine-readable service ID.",
+                        "type": "string",
+                        "x-order": 1
+                      },
+                      "service_name": {
+                        "description": "Service name.",
+                        "type": "string",
+                        "x-order": 2
+                      },
+                      "location_id": {
+                        "description": "Machine-readable location ID.",
+                        "type": "string",
+                        "x-order": 3
+                      },
+                      "location_name": {
+                        "description": "Location name.",
+                        "type": "string",
+                        "x-order": 4
+                      },
+                      "cron_expression": {
+                        "description": "How often backup will be run in cron format.",
+                        "type": "string",
+                        "x-order": 5
+                      },
+                      "start_time": {
+                        "description": "First backup wouldn't happen before this time.",
+                        "type": "string",
+                        "format": "date-time",
+                        "x-order": 6
+                      },
+                      "name": {
+                        "description": "Artifact name.",
+                        "type": "string",
+                        "x-order": 7
+                      },
+                      "description": {
+                        "description": "Description.",
+                        "type": "string",
+                        "x-order": 8
+                      },
+                      "retry_mode": {
+                        "description": "RetryMode specifies how backup should retry in case of failure.",
+                        "type": "string",
+                        "default": "RETRY_MODE_INVALID",
+                        "enum": [
+                          "RETRY_MODE_INVALID",
+                          "AUTO",
+                          "MANUAL"
+                        ],
+                        "x-order": 9
+                      },
+                      "retry_interval": {
+                        "description": "Delay between each retry.",
+                        "type": "string",
```

Comment:
why is this a string?

Reply:
I used `google.protobuf.Duration` as it used also on other places. And it needs more bits than JSON int can represent.
```
// # JSON Mapping
//
// In JSON format, the Duration type is encoded as a string rather than an
// object, where the string ends in the suffix "s" (indicating seconds) and
// is preceded by the number of seconds, with nanoseconds expressed as
// fractional seconds. For example, 3 seconds with 0 nanoseconds should be
// encoded in JSON format as "3s", while 3 seconds and 1 nanosecond should
// be expressed in JSON format as "3.000000001s", and 3 seconds and 1
// microsecond should be expressed in JSON format as "3.000001s".
```

Reply:
But will the API return only "s", or can it return "s", "m" and "h"?

Reply:
I think it should return only "s" based on that description